code/modules/spells/spell.dm

```diff
@@ -137,6 +137,8 @@
 
 // Where the cast chain starts
 /datum/action/cooldown/spell/PreActivate(atom/target)
+	if(SEND_SIGNAL(owner, COMSIG_MOB_ABILITY_STARTED, src) & COMPONENT_BLOCK_ABILITY_START)
+		return
```

Comment:
We're doing a null value return here, shouldn't we be doing `FALSE` instead to be nicer to consumers of `PreActivate`?